### DISCUSSION TITLE
Addition to Get started / Promt

### DISCUSSION THREAD

**Discussion by Chrisch3n** [JanDeDobbeleer/oh-my-posh#3064] [Ideas]
I had problems editing my PowerShell profile as per ["Get started" tutorial](https://ohmyposh.dev/docs/installation/prompt)
After a bit of search, I figured out, that I was unable to create/open the .ps1 file with the suggested commands because I was missing the base _.\PowerShell_ folder. When I manually added this folder under Documents, everything worked fine and the profile could even be created with the command "notepad $PROFILE"

Maybe you can add an information, in the [section](https://ohmyposh.dev/docs/installation/prompt) of the "Get started" guide, that this could be a problem and solution. 
If you are used to a shell, this might be obvious given the error, but it took a while for me.

````
PowerShell 7.3.0
PS C:\Users\Chris> New-Item -Path $PROFILE -Type File -Force
New-Item: Could not find a part of the path 'C:\Users\Chris\Documents\PowerShell\Microsoft.PowerShell_profile.ps1'.
````

Edit: Fixed Path in Error message, thanks to comment from @SiJiL82

**Comment by SiJiL82** [comment]
You've run the command in `C:\Users\Chris>` but it's trying to create the file in `C:\Users\Test` - a documents folder for a different user.
I suspect there's something wrong happened here, rather than the documentation needs updating.

**Reply by Chrisch3n** [reply]
Oh, Sorry, I changed the username for this post but of course I forgot to Edit it at both places. 
Both ran at C:\Users\Chris\... or better `New-Item: Could not find a part of the path 'C:\Users\Chris\Documents\PowerShell\Microsoft.PowerShell_profile.ps1'.` is the automatic error message responding to the `New-Iten` command for the specified location.
I will edit my original post to fix the error, but this wasn't the cause for the problem.

**Reply by JanDeDobbeleer** [reply]
@Chrisch3n it's really difficult for us to start explaining every detail for every external command. I understand the issue but it's a bit of a never ending street.

**Reply by Chrisch3n** [reply]
@JanDeDobbeleer Sure, I totally understand that mentioning every edge case can get out of hand quickly ;-) .
It was just a problem I noticed and wanted to share, in case it is a common error.
